extras/cronwrapper

```diff
@@ -14,11 +14,11 @@
 # LOGGING if true, logs all output to syslog daemon facility
 #         (typically /var/log/daemon.log or /var/log/syslog)
 #
-# Set CONFIGURED to true once you've setup the previous three
-# options.
+# NOTIFYONRETURN is used to define which return codes from plexupdate
```

Comment:
Actually, the way I did it did work as I wrote, you could list multiple codes and if RET hit one of them, you got a message. But I digress.